content/influxdb/cloud/tools/grafana.md

```diff
@@ -70,35 +70,74 @@ configure your InfluxDB connection:
 {{% tab-content %}}
 
 ## Configure Grafana to use InfluxQL
+To query InfluxDB Cloud from Grafana using InfluxQL:
 
-With **InfluxQL** selected as the query language in your InfluxDB data source,
-configure your InfluxDB connection:
+1. [Create an InfluxDB DBRP mapping](#create-an-influxdb-dbrp-mapping)
+2. [Configure your InfluxDB connection](#configure-your-influxdb-connection)
+
+### Create an InfluxDB DBRP mapping
+When using InfluxQL to query InfluxDB Cloud, the query must specify a database and a retention policy.
+Use the [`influx v1 dbrp create` command](/influxdb/cloud/reference/cli/influx/v1/dbrp/create/)
+command to create a database/retention policy (DBRP) mapping that maps a database
+and retention policy combination to an InfluxDB Cloud [bucket](/influxdb/cloud/reference/glossary/#bucket).
+
+{{% note %}}
+##### Automatically create DBRP mappings on write
+When using the InfluxDB 1.x compatibility API to write data to InfluxDB Cloud,
+InfluxDB Cloud automatically creates DBRP mappings for buckets whose names match the
+`db/rp` naming pattern of the database and retention policy specified in the write request.
+For more information, see [Database and retention policy mapping – Writing data](/influxdb/cloud/reference/api/influxdb-1x/dbrp/#when-writing-data).
+{{% /note %}}
+
+Provide the following:
+
+- database name
+- retention policy
+- [bucket ID](/influxdb/cloud/organizations/buckets/view-buckets/)
+
+```sh
+influx v1 dbrp create \
+  --db example-db \
+  --rp example-rp \
+  --bucket-id 00xX00o0X001 \
+  --default
+```
+
+_For more information about DBRP mapping, see [Database and retention policy mapping](/influxdb/cloud/reference/api/influxdb-1x/dbrp/)._
+
+### Configure your InfluxDB connection
+With **InfluxQL** selected as the query language in your InfluxDB data source settings:
 
 1. Under **HTTP**, enter the following:
 
-    - **URL**: Your [InfluxDB URL](/influxdb/cloud/reference/urls/).
+    - **URL**: Your [InfluxDB Cloud URL](/influxdb/cloud/reference/regions/).
 
         ```sh
-        https://cloud2.influxdata.com
+        https://cloud2.influxdata.com/
         ```
     - **Access**: Server (default)
 
-2. Under **Auth**, enable **Basic Auth**.
-3. Under **Basic Auth Details**, provide your InfluxDB authentication credentials:
+2. Under **Custom HTTP Headers**, select **Add Header**. Provide your InfluxDB Cloud authentication token:
```

Comment:
It's probably worth noting that I got this integration to work without using this section of the Grafana config at all.  Instead, I used my login email in the `User` field of "InfluxDB Details" and my token as the `Password` value.  

Is it worth calling out that either will work?  Is this unintended?

Reply:
I don't think it's "unintended," but I don't know if it's a path we want to document. Technically you can use basic auth with an arbitrary user name and your token as the password and it will work.